app/uk/gov/hmrc/exports/migrations/changelogs/cache/RenameAndConvertBorderTransportNationalityField.scala

```diff
@@ -88,3 +89,1062 @@ class RenameAndConvertBorderTransportNationalityField(countriesService: Countrie
     logger.info(s"Finished applying '${migrationInformation.id}' db migration.")
   }
 }
+
+object RenameAndConvertBorderTransportNationalityField {
+  private val countryMappings =
```

Comment:
Ah, I was thinking something more along the lines of `val allCountriesPlus = countriesService.allCountries ++ ("Norway" -> "NO")` rather than copying the whole countries file into the migration job. However this will work just as well and we'll be removing the migration jobs soon anyhow so not going to sweat it :wink:

Reply:
Oh yeah, that would have done it